src/Form/RenderFileTab.tsx

```diff
@@ -70,3 +71,29 @@ export default function RenderFileTab({ step: step, splitSchema, setSplitSchema
 export function FileTabComplete(step: Step) {
   return step.state.binary && step.state.code
 }
+
+export function RenderBasicFileTab({ step: step, splitSchema, setSplitSchema }: RenderInterface) {
```

Comment:
The `step: step` is pointless, you're renaming a variable to itself.

Reply:
Whoops! Fixed